Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -56,6 +56,8 @@ def audit_args
       switch "--display-filename",
              description: "Prefix every line of output with the file or formula name being audited, to "\
                           "make output easy to grep."
+      switch "--display-failures-only",
+             description: "Only display casks that fail the audit. This is the default for formulae."
```

Comment:
> This is the default for formulae.

Is this because there are no warnings for formulae or are warnings actually hidden by default for formulae?

Reply:
There are no warnings

Reply:
But there's also no message shown if the audit succeeds. It's either no output or a list of problems.